be/src/agent/agent_server.cpp

```diff
@@ -53,12 +55,8 @@ using std::vector;
 
 namespace doris {
 
-AgentServer::AgentServer(ExecEnv* exec_env,
-                         const TMasterInfo& master_info) :
-        _exec_env(exec_env),
-        _master_info(master_info),
-        _topic_subscriber(new TopicSubscriber()) {
-    
+AgentServer::AgentServer(ExecEnv* exec_env, const TMasterInfo& master_info)
+        : _exec_env(exec_env), _master_info(master_info), _topic_subscriber(new TopicSubscriber()) {
```

Comment:
This modification conflicts with Baidu's internal code style specifications, which requires the colon(':') to be placed after the previous line. Because all the code of the current Doris is like this, so I suggest to modify the format file to avoid modifying all the files.

Reply:
OK, I think set `BreakConstructorInitializers` to `AfterColon` will do the trick. Do you have any other suggestion?